.github/workflows/ci_frontend.yml

```diff
@@ -50,3 +50,44 @@ jobs:
           path: reports/jest-*.xml
           reporter: jest-junit
           working-directory: 'RetrospectiveExtension.Frontend'
+        
+      - if: steps.changes.outputs.src == 'true'
+        name: Get Coverage for badge
+        working-directory: 'RetrospectiveExtension.Frontend'
+        run: |
+          SUMMARY="$(npm test -- --coverageReporters='text-summary' | tail -2 | head -1)"
+          TOKENS=($SUMMARY)
+          # process.env.COVERAGE = '31.93%';
+          echo "COVERAGE=$(echo ${TOKENS[2]})" >> $GITHUB_ENV
+          # var REF = 'refs/pull/27/merge.json';
+          REF=${{ github.ref }}
+          # console.log('github.ref: ' + REF);
+          echo "github.ref: $REF"
+          # var PATHS = REF.split('/');
+          IFS='/' read -ra PATHS <<< "$REF"
+          # var BRANCH_NAME = PATHS[1] + PATHS[2];
+          BRANCH_NAME="${PATHS[1]}_${PATHS[2]}"
+          # console.log(BRANCH_NAME); // 'pull_27'
+          echo $BRANCH_NAME
+          # process.env.BRANCH = 'pull_27';
+          echo "BRANCH=$(echo ${BRANCH_NAME})" >> $GITHUB_ENV
+
+      - if: steps.changes.outputs.src == 'true'
+        name: Create the Badge
+        uses: schneegans/dynamic-badges-action@v1.0.0
+        with:
+          auth: ${{ secrets.GIST_SECRET }}
+          gistID: b414faf6b91eaaa6ec7d82882be36935
+          filename: retrotool__${{ env.BRANCH }}.json
+          label: Test Coverage
```

Comment:
I wonder if this label should be more specific, i.e. `Frontend Test Coverage`, since this is only test coverage for the React/Typescript portion of the extension.

Reply:
Will change the name accordingly